EIPS/eip-1193.md

```diff
@@ -338,6 +339,12 @@ Provider.request(args: RequestArguments): Promise<unknown>;
 The `request` method **MUST** send a properly formatted request to the Provider's Ethereum client.
 Requests **MUST** be handled such that, for a given set of arguments, the returned Promise either resolves with a value per the RPC method's specification, or rejects with an error.
 
+If the `args` object includes the `chainId` property, its value **SHOULD** be the integer ID of a chain as a hexadecimal string, per the [`eth_chainId`](https://eips.ethereum.org/EIPS/eip-695) Ethereum RPC method.
+
+The Provider **MAY** ignore the `chainId` property or interpret it in any manner whatsoever.
```

Comment:
This sounds a little too prescriptive, I wonder if this could just be:

```suggestion
This EIP does not specify how the Provider should interpret the `chainId` property or whether or not it will be handled at all.
```

Reply:
I also understand the word **MAY** is supposed to say the same thing, but I guess it just reads too strongly to me. Feel free to use your own judgment.